test/k8sT/microscope.go

```diff
@@ -56,14 +53,28 @@ var _ = Describe(microscopeTestName, func() {
 		microscopeErr, microscopeCancel := kubectl.MicroscopeStart()
 		Expect(microscopeErr).To(BeNil(), "Microscope cannot be started")
 
-		time.Sleep(time.Second * 10)
+		err := helpers.WithTimeout(func() bool {
+			res := kubectl.ExecPodCmd("kube-system", "microscope", "pgrep -f microscope")
+			return res.WasSuccessful()
+		}, "failed to start microscope",
```

Comment:
Error message is somewhat cryptic in relation to what this is checking for. I'd recommend something like "error finding running mircoscope processes" ( ? )